packages/hub/src/components/GlobalSearch/Snippet.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC, ReactNode } from "react";
+
+export const Snippet: FC<{ children: string }> = ({ children }) => {
+  // 'foo<b>bar</b>baz' -> ['foo', '<b>bar', '</b>baz']
+  const parts = children.split(/(?=<\/?b>)/);
+
+  const nodes: ReactNode[] = [];
+  let bold = false;
+  for (let part of parts) {
+    if (part.startsWith("<b>")) {
+      bold = true;
+      part = part.replace(/^<b>/, "");
+    } else if (part.startsWith("</b>")) {
+      bold = false;
+      part = part.replace(/^<\/b>/, "");
+    }
+    nodes.push(bold ? <strong>{part}</strong> : part);
+  }
```

Comment:
This is a very quick-and-dirty lexer, there might be a more elegant way to write this code, but it should work in most cases.

If the found text itself contains `<b>` as a string, the output probably will be messed up. If that ever happens, we can reconfigure `StartSel` and `StopSel` (see [Postgres docs](https://www.postgresql.org/docs/current/textsearch-controls.html#TEXTSEARCH-HEADLINE)) with more unique strings.

Reply:
I think it's good to start with something really basic here, we can make it more advanced later. Small PRs.